ui/src/components/RecordButton.js

```diff
@@ -46,6 +47,7 @@ export class RecordButton extends LitElement {
 
     updated() {
         this.handleWaveCanvas();
+        this.handleQCError();
```

Comment:
I feel like this function will be called very often unnecessarily--is there a certain property you could check if changed before calling it?

Reply:
```javascript
updated(changedProperties) {
  if (changedProperties.has('qcError') {
    this.handleQcError();
  }
}
```

https://lit-element.polymer-project.org/guide/lifecycle#updated